teradatalabs/centos6-ssh-oj8/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM jdeathe/centos-ssh:centos-6-1.2.0
+FROM jdeathe/centos-ssh:centos-6
```

Comment:
I think you can choose have that breakage or you can choose to have the breakage we're currently fixing. Your choice :-D